doc/notes/9283_poly_int.md

```diff
@@ -0,0 +1,121 @@
+---
+
+This work is licensed under the Creative Commons CC0 License
+
+---
+
+# Concrete Polymorphic Events
+### xtUML Project Implementation Note
+
+
+1. Abstract
+-----------
+
+2. Document References
+----------------------
+[1] [9283](https://support.onefact.net/redmine/issues/9283) Concrete Polymorphic Events  
+[2] [OOA 96 Report](http://www.ooatool.com/References.html) OOA 96 Report  
+[3] [OOA 97](http://www.ooatool.com/References.html) OOA 96  
+[4] Executable UML - Mellor and Balcer  
+[5] [9275](https://support.onefact.net/redmine/issues/9275) Incomplete Formalisation on Reflexive  
+
+3. Background
+-------------
+BridgePoint supports abstract polymorphic events as defined in [2] and [4].
+iUML supports concrete polymorphic events as described in [3].  This work
+moves to a strategy of concrete polymorphic events using a name-based
+strategy.  Inherited events are named with the MASL dotted syntax
+(super.eventName).  The presence of the dot in the name is the key to
+distinguishing events inherited from supertypes from events locally
+defined within the (present) subtype.
```

Comment:
Define some clear terms and use them consistently throughout the note. "Concrete polys" v. "Abstract polys" is the main one. They can read the literature but a concise definition would be helpful for future readers of this note. The biggest thing was it was confusing whether you were talking about actual polys (SM_PEVT) and concrete events (SM_SEVT) being used as polys